pkg/hierarchy/manager.go

```diff
@@ -0,0 +1,80 @@
+package hierarchy
```

Comment:
Please add the copyright notice. Same in other files.

Reply:
#2900